dotnet/src/InternalUtilities/planning/Extensions/ChatHistoryExtensions.cs

```diff
@@ -12,17 +12,17 @@ namespace Microsoft.SemanticKernel.Planning;
 /// <summary>
 /// Extension methods for <see cref="ChatHistory"/> class.
 /// </summary>
-internal static class ChatHistoryExtensions
+public static class ChatHistoryExtensions
```

Comment:
Does this need to be public?

Reply:
Probably not now that it's in internal utilities